_opensearch/rest-api/index-apis/put-mapping.md

```diff
@@ -36,23 +39,29 @@ PUT /<target-index>/_mapping
 PUT /_mapping
 ```
 
+You can also use the put mapping operation to update multiple indexes with one request.
+
+```
+PUT /<target-index1>,<target-index2>/_mapping
+```
+
 ## URL parameters
 
-All put mapping parameters are optional.
+All update mapping parameters are optional.
 
 Parameter | Data Type | Description
 :--- | :--- | :---
-&lt;target-index&gt; | Data Type | The index to add the mapping to. If you do not specify this parameter, OpenSearch adds the mapping to all indices within the cluster.
-allow_no_indices | Boolean | Whether to ignore wildcards that don’t match any indices. Default is `true`.
-expand_wildcards | String | Expands wildcard expressions to different indices. Combine multiple values with commas. Available values are `all` (match all indices), `open` (match open indices), `closed` (match closed indices), `hidden` (match hidden indices), and `none` (do not accept wildcard expressions), which must be used with `open`, `closed`, or both. Default is `open`.
-ignore_unavailable | Boolean | If true, OpenSearch does not include missing or closed indices in the response.
+&lt;target-index&gt; | N/A | Specifies an index to associate the mapping. If you do not specify this parameter, OpenSearch adds the mapping to all indexes within the cluster.
```

Comment:
"...to associate the mapping **with**."

Reply:
sure. "with which to associate the mapping"